docs/managing-data-quality/event-recovery-for-bdp-users/manual-event-recovery-for-snowplow-bdp/running/spark/index.md

```diff
@@ -1,9 +1,13 @@
 ---
 title: "Spark"
-date: "2020-08-26"
+date: "2020-04-14"
```

Comment:
Overwriting with @peel’s version. I checked that there were no relevant changes in the BDP version meanwhile.